code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -87,26 +87,56 @@
 		if(prob(5))
 			to_chat(src, "<span class = 'warning'>You are getting hungry!</span>")
 
-	find_nearby_disease()//getting diseases from
+	handle_body_temperature()//I bestow upon mice the gift of thermoregulation, so they can handle the fever caused by disease.
```

Comment:
how will this interact with mouseops?

Reply:
What are mouseops?

Reply:
Well if you wish mouseops can be made to not share diseases if we assume that their space suits are sterile.

Reply:
I think they are supposed to be temperature immune so they can fly through space in their "suits"

Make them behave consistent with however hardsuits behave with loose virii, if you can. Which I assume is "immune to airborne"

Reply:
both airborne and contact actually. ok I'll do it after the future rebase PR.